content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md

```diff
@@ -57,6 +57,10 @@ Kubernetes project, please check the installer tool's communication channels for
 information about what steps you need to take, and eventually if needed, follow up
 with maintainers to let them know about this change.
 
+The following diagram shows who's affected by this change in a visual form:
+
+{{< figure src="/blog/2023/08/31/legacy-package-repository-deprecation/flow.svg" alt="Visual explanation of who's affected by the legacy repositories being deprecated and frozen. Textual explanation is available above this diagram." >}}
```

Comment:
- For the version shown inline in the blog article, maybe we shouldn't tell them to go and follow a link to... this article?
- I'd set the class `diagram-small`, for more readonable scaling
- But I'd also make it **link** to a full-viewport version of the diagram so that folks can actually read it

---

- If you want to make the URL in the SVG itself clickable, great! (could be post merge)

Reply:
> For the version shown inline in the blog article, maybe we shouldn't tell them to go and follow a link to... this article?

I thought about this, but I'm not sure how to change the SVG to remove this.

> I'd set the class diagram-small, for more readonable scaling

I'll give this a try, thank for the suggestion!

> But I'd also make it link to a full-viewport version of the diagram so that folks can actually read it

Makes sense!

> If you want to make the URL in the SVG itself clickable, great! (could be post merge)

Let me try to do it prior to merge.

Reply:
I addressed all but the first comment, please take a look when you have some time. I think we can address the first comment in a follow up PR.